accounts/forms.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib.auth.forms import UserChangeForm
+from django.contrib.auth.models import User
+from django import forms
+
+
+class editProfileForm(UserChangeForm):
+    email = forms.EmailField(widget=forms.EmailInput(attrs={'class': 'form-control'}))
+    first_name = forms.CharField(max_length=100, widget=forms.TextInput(attrs={'class': 'form-control'}))
+    last_name = forms.CharField(max_length=100, widget=forms.TextInput(attrs={'class': 'form-control'}))
+    username = forms.CharField(max_length=100, widget=forms.TextInput(attrs={'class': 'form-control'}))
```

Comment:
Are you using the "form-control"? is it doing somthing?

Reply:
Deleted